CmfRoutingBundle.php

```diff
@@ -96,10 +96,21 @@ private function buildOrmCompilerPass(ContainerBuilder $container)
                     realpath(__DIR__.'/Resources/config/doctrine-orm') => 'Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Orm',
                 ),
                 array('cmf_routing.dynamic.persistence.orm.manager_name'),
-                'cmf_routing.backend_type_orm',
+                'cmf_routing.backend_type_orm_default',
                 array('CmfRoutingBundle' => 'Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Orm')
             )
         );
+
+        $container->addCompilerPass(
+            $doctrineOrmCompiler::createXmlMappingDriver(
+                array(
+                    realpath(__DIR__.'/Resources/config/doctrine-model') => 'Symfony\Cmf\Bundle\RoutingBundle\Model',
```

Comment:
Can't figure out why we need this? Is it not enough to just ensure the other pass doesn't get triggered?